src/build/arg/mod.rs

```diff
@@ -677,16 +675,17 @@ impl<'help> Arg<'help> {
     /// assert!(res.is_err());
     /// assert_eq!(res.unwrap_err().kind, ErrorKind::MissingRequiredArgument);
     /// ```
-    /// [`Arg::required_unless`]: ./struct.Arg.html#method.required_unless
-    /// [`Arg::required`]: ./struct.Arg.html#method.required
-    /// [`Arg::required_unless(name)`]: ./struct.Arg.html#method.required_unless
+    /// [required]: ./struct.Arg.html#method.required
     pub fn required_unless<T: Key>(mut self, arg_id: T) -> Self {
         self.r_unless.push(arg_id.into());
         self
     }
 
-    /// Sets args that override this arg's required setting. (i.e. this arg will be required unless
-    /// all these other arguments are present).
+    /// Sets this arg as [required] unless *all* of the specified arguments are present at runtime.
+    ///
+    /// In other words, parsing will succeed only if user either
+    /// * supplies the `self` arg.
```

Comment:
How about just saying `this arg`? I don't like how we are using `self` in the docs.

Reply:
`This arg` may refer to any of the args we're talking about, and it may not be clear to each one specifically. `The self arg` may not be very fluent, but it drives the point home even for those who don't know English well (but know Rust and what `self` means.)

Reply:
Ok, waiting for the tests to pass

Reply:
Sorry, I swear the test run on build server passed. Weird.